ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -73,6 +73,9 @@ def _get_axis_and_p(draw):
         helpers.dtype_values_axis(
             available_dtypes=helpers.get_dtypes("valid"),
             min_num_dims=2,
+            max_num_dims=4,  # limit maximum dimensions
+            min_dim_size=2,  # set minimum dimension size
+            max_dim_size=10,  # set maximum dimension size
```

Comment:
```suggestion
```
I don't think we need these to fix the tests.

Reply:
I have tested it we need max_num_dims=4 otherwise it is giving this error 

E           hypothesis.errors.InvalidArgument: ListStrategy(one_of(floats(min_value=-100.0, max_value=-3.097532938251384e-35, allow_nan=False, allow_infinity=False, allow_subnormal=False, width=32), floats(min_value=3.097532938251384e-35, max_value=100.0, allow_nan=False, allow_infinity=False, allow_subnormal=False, width=32)), min_size=45_000, max_size=45_000) can never generate an example, because min_size is larger than Hypothesis supports.  Including it is at best slowing down your tests for no benefit; at worst making them fail (maybe flakily) with a HealthCheck error.
E           while generating 'args' from _get_axis_and_p()
E